src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -70,6 +70,14 @@ public function supportsNormalization($data, $format = null)
         return \is_object($data) && !$data instanceof \Traversable;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function varyBy(): array
+    {
+        return array('type');
```

Comment:
shouldn't we use short array notation here, as PHP 7.1 is the min version for 4.x?

Reply:
Nop, this has been discussed numerous times already and we keep the sort array notation for consistency 🙃

Reply:
(/s/short/long/)

Reply:
It'll be reconsidered at last when 3.4 will be EOLed I suppose.

Reply:
when 2.8 gets EOLed. 3.4 is not an issue. We can migrate it as it supports PHP 5.5+

Reply:
Indeed!